### ISSUE TITLE
[PC] - Controller remapping + Unable to map controller to "touchscreen"

### ISSUE THREAD

**Issue by AuricSilver** [streetpea/chiaki-ng#372]
**Is your feature request related to a problem? Please describe.**
Yes.

When using an Xbox controller on PC with Chikai4Deck, there is no button mapping on the controller to press down on the touchscreen.  This is an important feature in multiple games, for things including pulling up maps, pausing, and other various functions within games.  

When entering the key remapping feature of Chiaki, buttons also do not reflect or change to address xinput controllers such as the Xbox controller line.  The right stick cannot be mapped, among other buttons on the controller entirely.  This needs to be addressed quickly, for a game's full functionality cannot be reached on an Xbox controller.

When remapping the controller, 

**Describe the solution you'd like**

-The key remapping feature to include controller mappings and button prompts
-The key remapping feature to include the ability to recognize all buttons on the Xbox series of controllers
-(Optional) the Share Button to no longer be mapped to the share button by default, but instead the press of the touchscreen; I find share buttons on controllers to be functionally useless, but others may disagree.

**Describe alternatives you've considered**
-None currently; different controllers (especially the Xbox and Switch Pro controllers) should be taken into heavy consideration moving forward, especially when it comes to basic remapping options.

**Additional context**
I'm not sure what I can provide to this, but I would appreciate this issue is ultimately looked into.  Thank you so much for listening to me.

**Comment by streetpea** [comment]
@AuricSilver you can always add Chiaki4deck to Steam using the button on the main menu and then make the controls using Steam input to the correct keyboard inputs.

**Comment by AuricSilver** [comment]
That's fair and fine, @streetpea , but native support would be appreciated regardless.  Still, thank you for telling me about this.

**Comment by SeanWangTWs** [comment]
Can we just run chiaki4deck directly and remap the touch screen button to the xbox one controller without steam input?

**Comment by fbermudez231** [comment]
Glad to know I am not the only one trying to get this feature done somehow.
I have been trying to compile previous chiaki after change the 363 line of  \chiaki-v2.2.0\gui\src\controllermanager.cpp
to
case SDL_CONTROLLER_BUTTON_BACK:
ps_btn = CHIAKI_CONTROLLER_BUTTON_TOUCHPAD;

 using msys2 on windows. But I keep failing to get the app to compile with the next error:
LINK : fatal error LNK1104: cannot open file 'LIBCMT.lib'
I have been switching MSVC tools to try to figure out the right one, but after try so many I even lost the track of the ones I tested already. This was the latest version I tested: 14.29.30133.
Is it possible to someone to point me to the right one, any tips are welcome?
Thanks for all the good work getting this app more accessible.
